tests/test_actions.py

```diff
@@ -1409,14 +1409,35 @@ def test_defaults(self, mocked_webdriverwait, mocked_ec, Tester) -> None:
 
         Wait.for_the(test_target).perform_as(Tester)
 
-        mocked_webdriverwait.assert_called_once_with(mocked_browser, settings.TIMEOUT)
+        mocked_webdriverwait.assert_called_once_with(
+            mocked_browser, settings.TIMEOUT, settings.POLLING
+        )
         mocked_ec.visibility_of_element_located.assert_called_once_with(test_target)
         mocked_webdriverwait(
             mocked_browser, settings.TIMEOUT
         ).until.assert_called_once_with(
             mocked_ec.visibility_of_element_located(test_target.locator)
         )
 
+    @mock.patch("screenpy_selenium.actions.wait.EC", autospec=True)
+    @mock.patch("screenpy_selenium.actions.wait.WebDriverWait", autospec=True)
+    def test_override(self, mocked_webdriverwait, mocked_ec, Tester) -> None:
+        test_target = Target.the("foo").located_by("//bar")
+        mocked_ec.visibility_of_element_located.__name__ = "foo"
+        mocked_browser = get_mocked_browser(Tester)
+
```

Comment:
My nitpicking comb is set to its _finest_ setting to tell you that this blank line should be gone because `timeout` is part of the setup. :P

Reply:
fixed